### PR TITLE
run.tests

### PR DESCRIPTION
**Pull request by sidharthv96**

### CODE REVIEW

File: __mocks__/@actions/github/index.ts [hunk 1]
```diff
@@ -10,4 +10,10 @@ export const clearContextMock = () => {
     });
 };
 
-export const getOctokit = jest.fn();
+export const getOctokit = jest.fn().mockReturnValue({
```
Comment (ArtiomTr) [inline __mocks__/@actions/github/index.ts:13]: Calling `mockReturnValue` here will mock the value for all tests, which can be confusing for other people. Instead, you can call it directly in your test. For example:

```ts
import { getOctokit } from '@actions/github';

(getOctokit as jest.Mock<any, any>).mockReturnValue({ /* your value */ });

```

**Review by ArtiomTr — CHANGES_REQUESTED** [review]
Everything looks good, only one change needs to be done ✨

File: __mocks__/@actions/github/index.ts (hunk 1 above)
Comment (sidharthv96) [reply __mocks__/@actions/github/index.ts]: Done. :)

**Review by ArtiomTr — APPROVED** [review]
Perfect 👍

**Pull request merged**